scripts/upgradeable/upgradeable.patch

```diff
@@ -329,6 +329,36 @@ index 304d1386a..a1cd63bee 100644
 -@openzeppelin/contracts/=contracts/
 +@openzeppelin/contracts-upgradeable/=contracts/
 +@openzeppelin/contracts/=lib/openzeppelin-contracts/contracts/
+diff --git a/test/account/AccountERC7702.test.js b/test/account/AccountERC7702.test.js
+index d08a52209..7a44bccfe 100644
+--- a/test/account/AccountERC7702.test.js
++++ b/test/account/AccountERC7702.test.js
+@@ -26,8 +26,8 @@ async function fixture() {
+ 
+   // domain cannot be fetched using getDomain(mock) before the mock is deployed
+   const domain = {
+-    name: 'AccountERC7702Mock',
+-    version: '1',
++    name: '', // Not initialized in the context of signer
++    version: '', // Not initialized in the context of signer
```

Comment:
As part of the upgradeable transpilation pipeline, we generate the upgradeable versions and the run tests against them. However, ERC-7702 accounts will delegate to an EIP-712 (through ERC7739) contract, so the domain separator [will be rebuilt each time](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/aaf955de5664a23e6180c1887bf363ab24f2054e/contracts/utils/cryptography/EIP712.sol#L86) without the `name` or `version` set in the EOA's storage.

In practice, if users are delegating to an upgradeable contract, it should be a proxy pointing to an implementation that uses the EIP712Upgradeable version so that the domain can be initialized (and reinitialized) properly.